CONTRIBUTING.md

```diff
@@ -2,84 +2,110 @@
 
-#### Update the NPM Version 
+Note that at this time, this only works for React components, not themes or other imports.
 
-Open the [shared-components app](https://shared-components.glitch.me) on Glitch, then
+You can see a demo of this in `/test/remote-component/index.js`.
+Create a remix of shared-components ([remix link](https://glitch.com/edit/#!/remix/shared-components)) and make all your changes within the remix .
 
+Note:
+- this does not currently work with SSR in the community site. The current work around is to go in Glitch-Community/server/routes.js and change this line:
 ```
-# 1. Pull the latest changes from Github 
-git pull origin master
-
-# 2. Update the NPM version
-npm version patch
-
-# 3. Update changelog.md with your changes
+const renderPage = require('./render');
+```
+to
+```
+const renderPage = () => ({ html: null, helmet: null, styleTags: null });
+```
+Another note: if you plan on using this component in multiple places you'll have to add the createRemoteComponent every place the component is used. This might be tricky if you're editing a component that's used widely, so you may wish to try the following process below instead.
 
-# 4. Add your changes
-git add -A
-git commit -m "your commit message"
+#### Testing a version of shared-components in the community stie
+You may decide you'd rather work locally or you'd rather test an entire version of shared-components before publishing. There are a number of different ways to do this but here is one possible route that may work for you:
 
-# 5. Push your changes to Github
-git push origin master
+1. cd into community
+1. install your local copy of shared-componnents [using npm's local file path's feature](https://docs.npmjs.com/files/package.json#local-paths) ex: `npm install ../shared-components`
+1. in community/aliases.js add to your webpack aliases: 
 ```
-
-Finally, you can choose to either publish the changes as release candidate so you can test it within a ~community remix
+  react: path.resolve('./node_modules/react'),
+  'styled-components': path.resolve('./node_modules/styled-components')
 ```
-# 1. Locally, pull the latest changes from master
+We do this because otherwise shared-components will use its own version of react and styled-components, and you'll get an [error message mentioning invalid hooks](https://reactjs.org/warnings/invalid-hook-call-warning.html). In theory any package shared-components uses that is also used by community will need a similar alias to ensure we're only using one dependency. Note: this process that was just outlined is kind of a pain and not particularly sustainable. If you find a better alternative feel free to update this documentation!
```

Comment:
good clarification I'll fix thanks! 

When you say prepublish are you referring to the pre-release thing? I def can mention it, I think the benefits of a prerelease are not that clear to me vs just publishing a regular release. I'm guessing that it reduces the risk of accidentally upgrading to a broken version?

Reply:
Yeah, prepublish = prerelease. 
I think the benefits of a pre-release version are that you don't have to mess around with setting up a local npm link, and it's perhaps easier for other folks to test out your changes. (You might say, hey teammate, change the line in package.json to say `X.XX.XX-pre`, instead of saying 'check out this branch of shared-components and then build it and then do the npm link thing')

Reply:
cool cool will update